components/header.js

```diff
@@ -128,6 +121,30 @@ class Header extends HTMLElement {
             height: 22.5px;
           }
         }
+
+        &:focus {
+          outline: 3px solid #846DE9;
+          outline-offset: 2px;
+          border-radius: 10px;
+        }
+
+        &:focus-visible {
+          outline: 3px solid #24EACA;
```

Comment:
혹시 Prettier를 꺼놓으셨나요? 🤨

```suggestion
          outline: 3px solid #24eaca;
```